src/lib/arch/aarch64/ExceptionHandler.cc

```diff
@@ -538,6 +543,14 @@ bool ExceptionHandler::init() {
             ChangeType::REPLACEMENT, {R0}, {static_cast<uint64_t>(result)}};
         break;
       }
+      case 210: {  // shutdown
+        // TODO: Functionality omitted - returns -38 (errno 38, function not
+        // implemented) The targetted linux kernel does not implement this
```

Comment:
Just reword to:

TODO: Functionality omitted - returns -38 (errno 38, function not implemented). The targetted linux kernel does not implement this syscall.